src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/ActionBar.xaml

```diff
@@ -75,69 +75,93 @@
+                Background="Transparent"
+                Style="{StaticResource SubtleButtonStyle}"
                 Tapped="PrimaryButton_Tapped"
                 Visibility="{x:Bind ViewModel.PrimaryAction.Name, Converter={StaticResource StringNotEmptyToVisibilityConverter}, Mode=OneWay}">
 
                 <StackPanel Orientation="Horizontal" Spacing="8">
-                    <cpcontrols:IconBox
-                        Width="16"
-                        Height="16"
-                        Margin="4,4,4,4"
-                        SourceKey="{x:Bind ViewModel.PrimaryAction.Icon, Mode=OneWay}"
-                        SourceRequested="{x:Bind help:IconCacheProvider.SourceRequested}" />
-
-                    <StackPanel Orientation="Vertical" Spacing="2">
-                        <TextBlock
-                            FontSize="12"
-                            Style="{StaticResource CaptionTextBlockStyle}"
-                            Text="{x:Bind ViewModel.PrimaryAction.Name, Mode=OneWay}" />
+                    <TextBlock
+                        VerticalAlignment="Center"
+                        Style="{StaticResource CaptionTextBlockStyle}"
+                        Text="{x:Bind ViewModel.PrimaryAction.Name, Mode=OneWay}" />
+                    <Border
+                        Padding="4"
+                        Background="{ThemeResource SubtleFillColorSecondaryBrush}"
+                        BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}"
+                        BorderThickness="1"
+                        CornerRadius="4">
                         <FontIcon
                             HorizontalAlignment="Left"
                             VerticalAlignment="Center"
                             FontSize="10"
                             Foreground="{ThemeResource TextFillColorSecondaryBrush}"
                             Glyph="&#xE751;" />
-                    </StackPanel>
+                    </Border>
                 </StackPanel>
             </Button>
             <Button
                 x:Name="SecondaryButton"
-                Height="40"
-                Padding="8,4,8,4"
+                Padding="6,4,4,4"
                 x:Load="{x:Bind IsLoaded, Mode=OneWay}"
+                Style="{StaticResource SubtleButtonStyle}"
                 Tapped="SecondaryButton_Tapped"
                 Visibility="{x:Bind ViewModel.HasSecondaryCommand, Mode=OneWay}">
-                <StackPanel Orientation="Horizontal" Spacing="8">
-                    <cpcontrols:IconBox
+
+                <!--<cpcontrols:IconBox
                         Width="16"
                         Height="16"
                         Margin="4,4,4,4"
                         SourceKey="{x:Bind ViewModel.SecondaryAction.Icon, Mode=OneWay}"
-                        SourceRequested="{x:Bind help:IconCacheProvider.SourceRequested}" />
+                        SourceRequested="{x:Bind help:IconCacheProvider.SourceRequested}" />-->
```

Comment:
we can just delete it entirely to appease spellbot

Reply:
Done!